crates/cli/templates/main.rs

```diff
@@ -0,0 +1,10 @@
+#![cfg_attr(target_os = "zkvm", no_main)]
```

Comment:
look at the book's quickstart: for starter templates we use std and enable the "std" feature now